ssl/statem/statem_clnt.c

```diff
@@ -1348,6 +1348,10 @@ static int set_client_ciphersuite(SSL *s, const unsigned char *cipherchars)
         if (SSL_IS_TLS13(s)) {
             const EVP_MD *md = ssl_md(s->ctx, c->algorithm2);
 
+            if (!ossl_assert(s->esssion->cipher != NULL)) {
```

Comment:
Actually there is a typo here. Should be `session`

Reply:
Yes, fixed to correct s->session->cipher